### ISSUE TITLE
White corners and transitions when using a dark theme on a dialog

### ISSUE THREAD

**Issue by AdamMYoung** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#1043]
When applying the dark theme to a user control for use in a DialogHost, the transitions and corners display as the light theme rather than the dark theme.

Theme is applied like so:
```<UserControl.Resources>
        <ResourceDictionary
            Source="pack://application:,,,/MaterialDesignThemes.Wpf;component/Themes/MaterialDesignTheme.Dark.xaml" />
    </UserControl.Resources>
```

![Pic](https://i.imgur.com/rBpDQ4j.png)

My application is here if you need a source: https://github.com/AdamMYoung/Groupr

Thanks

**Comment by Keboo** [comment]
@AdamMYoung I believe this will be addressed with the upcoming 2.5.0 release. Specifically the change from #1036. Can you test with the latest 2.5.0 preview nuget package, try setting the new `DialogHost.DialogTheme` property to `Dark` and confirm the problem is resolved?

**Comment by AdamMYoung** [comment]
@Keboo Perfect, working as intended now. Thanks
